### ISSUE TITLE
Oxygen Skin or Styling

### ISSUE THREAD

**Issue by JFOC** [ERROPiX/hydrogen-pack#15]
Adding a skin for oxygen editor or font styling

**Comment by JFOC** [comment]
> This means something like EEP UI skins?

yes something like in EEP

**Comment by HenkDz** [comment]
Good idea but we better focus on the functionality first. We want power over beauty.
